cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -634,11 +634,12 @@ public QueryHandle createNamespace(Id.Namespace namespace) throws ExploreExcepti
       Map<String, String> sessionConf = startSession();
       SessionHandle sessionHandle = cliService.openSession("", "", sessionConf);
 
+      String database = getHiveDatabase(namespace.getId());
       // "IF NOT EXISTS" so that this operation is idempotent.
-      String statement = String.format("CREATE DATABASE IF NOT EXISTS %s", getHiveDatabase(namespace.getId()));
+      String statement = String.format("CREATE DATABASE IF NOT EXISTS %s", database);
```

Comment:
unrelated to this PR, but shouldn't it fail if the db exists? You should only be able to create the DB once - during namespace creation, no?